src/boot/stylesheets/actions.scss

```diff
@@ -75,34 +75,44 @@
 	&:only-child {
 		padding: 0 18px;
 	}
-
-	a {
-		outline: none;
-	}
-
-	a.inactive-action:hover {
-		color: $orange-jazzy;
-	}
-
-	a.active-action:hover {
-		color: $gray-text-min;
-	}
 }
 
 .wpnc__main div.wpnc__action-link {
+	cursor: pointer;
 	text-align: center;
 	font-size: 12px;
+	height: 55px;
```

Comment:
Curious, but why do we need to set height here? Did we used to have a default height on links?